install_config/registry/extended_registry_configuration.adoc

```diff
@@ -82,40 +82,40 @@ unnecessary downtime for pods and does not actually clear the cache.
 [[whitelisting-docker-registries]]
 == Whitelisting Docker Registries
 
-You can specify a whitelist of docker registries, allowing you to curate a set
-of images and templates that are available for download by {product-title}
-users. This curated set can be placed in one or more docker registries, and then
-added to the whitelist. When using a whitelist, only the specified registries
-are accessible within {product-title}, and all other registries are denied
-access by default.
+You can specify a whitelist of Docker registries, allowing you to curate a set 
+of images and templates that are available for download by {product-title} users.
+
+This curated set can be placed in one or more docker registries, and then added 
+to a whitelist defined in the *_/etc/containers/registries.conf_* file. 
+
+When using a whitelist, you can specify which secure and insecure registries 
+a user is allowed to download from, and which registries users cannot download
+from. 
 
 To configure a whitelist:
 
-. Edit the *_/etc/sysconfig/docker_* file to block all registries:
+. Edit the *_/etc/containers/registries.conf_* file to add or edit the following
+parameters as needed:
 +
 ----
-BLOCK_REGISTRY='--block-registry=all'
-----
-+
-You may need to uncomment the `*BLOCK_REGISTRY*` line.
-. In the same file, add registries to which you want to allow access:
-+
-----
-ADD_REGISTRY='--add-registry=<registry1> --add-registry=<registry2>'
-----
-+
-.Allowing Access to Registries
-----
-ADD_REGISTRY='--add-registry=registry.redhat.io'
-----
-+
-This example would restrict access to images available on the
-link:https://access.redhat.com/search/#/container-images[Red Hat Customer Portal].
+[registries.search] <1>
+registries = ["reg1.example.com", "reg2.example.com"]
+
+[registries.insecure] <2>
+registries = ["reg3.example.com"] 
 
-Once the whitelist is configured, if a user tries to pull from a container image registry
-that is not on the whitelist, they will receive an error message stating that
-this registry is not allowed.
+[registries.block] <3>
+registries = ['docker.io']
+----
+<1> Specify the secure registries that users can download images.
```

Comment:
The `registries.search` list is the list of registry servers the client (docker or cri-o in this situation) should search against when only an image's short name is specified.  For example if `registries.search` is `['registry.redhat.io','docker.io','registry.example.com']` and the image `myimage:latest` is specified, then the client will start trying to pull from the search registries in the order they're listed (`registry.redhat.io/myimage:latest`, `docker.io/myimage:latest`...) until the image is finally found.  For search it does not matter if the registry is secure or not.  How the client tries to communicate with an secure/insecure registry is handled by `registry.insecure`.